server/config_example.md

```diff
@@ -8,6 +8,7 @@
 
 # Location
 message_outreach_radius = 100 # meters
+message_retrival_limit = 300
```

Comment:
Can we change this 1 or 0. This means that every single time a new message is added it pull the most recent **300** messages.